homeassistant/components/sql/util.py

```diff
@@ -0,0 +1,12 @@
+"""Utils for sql."""
+from __future__ import annotations
+
+from homeassistant.components.recorder import get_instance
+from homeassistant.core import HomeAssistant
+
+
+def resolve_db_url(hass: HomeAssistant, db_url: str | None) -> str | None:
```

Comment:
```suggestion
def resolve_db_url(hass: HomeAssistant, db_url: str | None) -> str:
```
db_url is not optional on an instance of Recorder. If this is removed, tests will hopefully pass.